iceoryx_utils/test/moduletests/test_relative_pointer.cpp

```diff
@@ -18,20 +18,20 @@
 #include "iceoryx_utils/platform/stat.hpp"
 #include "iceoryx_utils/platform/unistd.hpp"
 
-
 #include "test.hpp"
 
 #include <cstdint>
 #include <cstring>
+#include <fcntl.h> /* For O_* constants */
```

Comment:
I think all the changes in this file are due to merge errors

Reply:
Fixed.